Step  3

AugustoCabrera opened the pull request:
Cambios con Unitest y Actions incorporado.

AugustoCabrera committed:
avances de Step 1

AugustoCabrera committed:
fin Step 1

AugustoCabrera committed:
Agrego y linkeo librerias

AugustoCabrera committed:
Unitest terminados

AugustoCabrera committed:
Actions fin

AugustoCabrera committed:
Pull Request funciona

AugustoCabrera committed:
CL:

Prueba llave

GabrielEValenzuela reviewed (changes requested):
Add missing end lines

Don't include build files.

Missing documentation

AugustoCabrera committed:
Cambios Archivos Eliminados y runLab.sh agregado

AugustoCabrera committed:
Modifications:

- Comments written in English.

- Add missing end lines

- Delete build files.

- Add documentation

- Fix warnings in static.c and shared.c

- Incorporate a sh (runLab.sh) to automate execution.

AugustoCabrera commented:
Modifications:

- Comments written in English.
- Add missing end lines
- Delete build files.
- Add documentation
- Fix warnings in static.c and shared.c
- Incorporate a sh (runLab.sh) to automate execution.

Soon I will incorporate CDash and Sonnarqube!!
